dandi/tests/test_organize.py

```diff
@@ -249,3 +250,44 @@ def error_link(src, dest):
     monkeypatch.setattr(os, "symlink", succeed_link if sym_success else error_link)
     monkeypatch.setattr(os, "link", succeed_link if hard_success else error_link)
     assert detect_link_type(tmp_path) == result
+
+
+@pytest.mark.parametrize("mode", ["copy", "move"])
+@pytest.mark.parametrize("video_mode", ["copy", "move", None])
+def test_video_organize(video_mode, mode, create_video_nwbfiles):
+    dandi_organize_path = create_video_nwbfiles.parent / "dandi_organized"
+    dandi_organize_path.mkdir(parents=True, exist_ok=True)
+    cmd = [
+        "--files-mode",
+        mode,
+        "--rewrite",
+        "external-file",
+        "--external-files-mode",
+        video_mode,
```

Comment:
When `video_mode` is `None`, both it and the `"--external-files-mode"` string need to be omitted from the command.

Reply:
https://github.com/dandi/dandi-cli/pull/841/commits/37e951e6f02fd159ebbedee2c5d759603fce0a7e